guides/common/modules/proc_importing-into-the-library-environment.adoc

```diff
@@ -1,5 +1,5 @@
 [id="Importing_into_the_Library_Environment_{context}"]
-= Importing into the Library Environment
+= Importing into the Library environment
```

Comment:
looking at this: Do we need a very short (!) list of items that are capitalized? 

* Library
* Content View
* Names like Ansible, Puppet, and Salt
* ?
* Kickstart

Reply:
Do you think we need a list? Is there a problem with new contributors not knowing what to capitalize?

Reply:
Yes, I am one of those people. When working locally, I just use `rg` and go with the most results. I think a handy small list would be neat.

Reply:
We could add a list to the Contributing section. I can create an issue for this.

Reply:
Looks like Aneta created an [issue](https://github.com/theforeman/foreman-documentation/issues/2553) to change Content View to lc, so that will not be on our list.